zubhub_frontend/zubhub/src/views/project_details/ProjectDetails.jsx

```diff
@@ -443,7 +450,52 @@ function ProjectDetails(props) {
           }
           aria-labelledby={t('projectDetails.ariaLabels.imageDialog')}
         >
-          <img className={classes.enlargedImageStyle} src={enlarged_image_url} alt={`${project.title}`} />
+          <IconButton
+            className={classes.cancelEnlargedImageBtn}
+            onClick={() =>
+              setState({
+                ...state,
+                open_enlarged_image_dialog: !open_enlarged_image_dialog,
+              })
+            }
+            aria-label={t('projectDetails.ariaLabels.closeImageDialog')}
+          >
+            <CloseIcon
+              className={classes.enlargedImageDialogCloseIcon}
+            />
+          </IconButton>
+          <Box className={classes.enlargedImageContainer}>
+            {project.images.length <= 1 ? (
+              <img
+                className={classes.enlargedImageStyle}
+                src={enlarged_image_url}
+                alt={`${project.title}`}
+              />
+            ) : (
+                <Slider
+                  initialSlide={enlarged_image_id}
+                  adaptiveHeight
+                  infinite
+                  speed={500}
+                  slidesToShow={1}
+                  slidesToScroll={1}
+                  className={classes.enlargedImageStyle}
+                  nextArrow={<EnlargedImgArrow />}
+                  prevArrow={<EnlargedImgArrow />}
+                >
+                  {project.images.map(image => (
+                    <div>
+                      <img
+                        className={classes.sliderImageStyle}
+                        key={image.public_id}
+                        src={image.image_url}
+                        alt={''}
```

Comment:
@tuxology I think we should ask optional image descriptions while uploading images for accessibility